### PR TITLE
ink client + better structure

### PR DESCRIPTION
**Pull request by LawlAoux**

### CODE REVIEW

File: config_to_bot/examples/pizza/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,374 @@
+{
```
Comment (uriva) [inline config_to_bot/examples/pizza/package-lock.json:1]: i think you didn't mean to add this file

File: config_to_bot/examples/hello/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,374 @@
+{
```
Comment (uriva) [inline config_to_bot/examples/hello/package-lock.json:1]: i think you didn't mean to add this file

File: config_to_bot/debugger/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,8287 @@
+{
```
Comment (uriva) [inline config_to_bot/debugger/package-lock.json]: i think you didn't mean to add this file

File: config_to_bot/debugger/cli.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
```
Comment (uriva) [inline config_to_bot/debugger/cli.js:1]: is this line required?

File: config_to_bot/debugger/.gitattributes [hunk 5]
```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```
Comment (uriva) [inline config_to_bot/debugger/.gitattributes:1]: i think you didn't mean to add this file

File: config_to_bot/debugger/.editorconfig [hunk 6]
```diff
@@ -0,0 +1,12 @@
+root = true
```
Comment (uriva) [inline config_to_bot/debugger/.editorconfig:1]: i think you didn't mean to add this file

**Comment by uriva** [comment]
the .DS file seems to have been added by accident

File: config_to_bot/debugger/readme.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# ink_client_for_examples
```
Comment (uriva) [inline config_to_bot/debugger/readme.md:1]: i think you didn't mean to add this file

File: config_to_bot/debugger/test.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (uriva) [inline config_to_bot/debugger/test.js:1]: i think you didn't mean to add this file

File: config_to_bot/debugger/ui.js [hunk 9]
```diff
@@ -0,0 +1,67 @@
+import WebSocket from "ws";
```
Comment (uriva) [inline config_to_bot/debugger/ui.js:1]: the indent here seems excessive, we usually use 2 spaces for js

**Review by uriva — APPROVED** [review]

File: config_to_bot/debugger/ui.js (hunk 9 above)
Comment (LawlAoux) [reply config_to_bot/debugger/ui.js:1]: I am not sure what you mean, I used a formatter in this file

File: config_to_bot/main.py [hunk 10]
```diff
@@ -28,16 +28,16 @@ async def responder_with_state(request):
...
+                return {"botUtterance": "Starting Over"}
             try:
-                computation_result = await bot({composers.event: request, **state})
+                computation_result = await bot(state, {composers.event: request})
                 state = computation_result
-                return (
-                    state[graph.make_computation_node(composers.utter)],
-                    gamla.pipe(
+                return {
+                    "botUtterance": state[graph.make_computation_node(composers.utter)],
```
[the remark this answers is not in the document]
Comment (LawlAoux) [reply config_to_bot/main.py:39]: Will do this after the demo

File: config_to_bot/debugger/ui.js (hunk 9 above)
Comment (uriva) [reply config_to_bot/debugger/ui.js:1]: can you switch your formatter to prettier, without any configurations?

**Pull request merged**